ref/general/authentication.adoc

```diff
@@ -34,18 +38,30 @@ Open Liberty supports the following options for SSO authentication:
 
+- link:/docs/ref/general/#/docs/concept/sso-config-json.html[JSON Web Tokens] +
+JSON Web Tokens (JWT) are used to propagate the user identity that is established by Single sign-on (SSO) between different microservice applications. By configuring SSO user authentication with JWT, you can manage the identity that is shared between applications.
+
 - link:/docs/ref/general/#/docs/concept/sso-config-ltpa.html[LTPA cookies] +
 Lightweight Third Party Authentication (LTPA) provides SSO configuration support to authenticate users when they are accessing application servers. With LTPA enabled, you can encrypt, digitally sign, and securely transmit authentication-related data between servers.
 
 - link:/docs/ref/feature/#openidConnectServer-1.0.html[OpenID Connect]  +
-OIDC is a popular internet SSO protocol that works well with cloud-native applications. This protocol enables a client application to request the identity of the user as an ID token in a standardized, REST-like manner. The client application, or Relying Party, requests information about a user from an authorization server, known as an OpenID Connect Provider. After users are authenticated by an OpenID Provider, they can access any Relying Party that is configured for that Provider, without having to manage unique accounts for each application or website.
+OIDC is a popular internet SSO protocol that works well with cloud-native applications. This protocol enables a client application to request the identity of the user as an ID token in a standardized, REST-like manner. The client application, or Relying Party, requests information about a user from an authorization server, which is known as an OpenID Connect Provider. After users are authenticated by an OpenID Provider, they can access any Relying Party that is configured for that Provider, without having to manage unique accounts for each application or website.
 
 - link:/docs/ref/general/#/docs/concept/sso-config-saml.html[Security Assertion Markup Language (SAML 2.0)] +
 SAML Web Browser SSO enables web applications to delegate user authentication to a SAML identity provider instead of a configured user registry.
 
 - link:/docs/ref/feature/#spnego-1.0.html[Simple and Protected GSS-API Negotiation Mechanism (SPNEGO)] +
 With SPNEGO SSO, users log in to the Microsoft Domain controller once to access and move between protected resources on Open Liberty servers without getting prompted for additional logins.
 
+= JAAS authentication with Open Liberty
+
+Open Liberty relies on the Java Authentication and Authorization Service (JAAS) framework to determine a user's identity and access roles. JAAS authentication is suited to microservice-based applications because it separates the authentication process from the rest of the application so that authentication can be managed independently. New or updated authentication processes can be plugged in to an application without otherwise modifying the application itself.
```

Comment:
This doesn't make sense to me.

Reply:
there wasn't much conceptual info for JAAS in the KC resources so I went back to the Oracle spec to try and draft an overview of why JAAS should matter to  microservice developers using OL. I may have mis-conflated some aspects in the process. Also probably why this section seems out of joint with the rest. There was originally a ref to JAAS as "pluggable" here but a reviewer asked me to remove it. Combining/simplifying these two pps in this commit